### DISCUSSION TITLE
RTL support for lottie internally

### DISCUSSION THREAD

**Discussion by wsdwsd0829** [airbnb/lottie-ios#1695] [General]
1. Just checking if this can be support from within the framework? 

2. I am currently relying on update the transform for LOTAnimationView. 
Any suggestions besides that, like what other alternatives I can achieve that. 

3. What if there is a layer that I want to filp in rtl but not for another layer?

**Comment by calda** [comment]
Flipping the layer transform sounds like a good approach. If you need to flip some layers but not others, it may make the most sense to have two different copies of the animation (one for LTR and one for RTL).
